### PR TITLE
[Hosted Elastic Agent] Add info about auto unenrollment

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc [hunk 1]
```diff
@@ -511,3 +511,10 @@ installed integrations.
 
 . Under advanced settings, set `windows.advanced.diagnostic.enabled`
 to `false`, and then save the integration.
+
+
+[discrete]
+[[hosted-agent-offline]]
+== Hosted {agent} Offline
+
+In Elastic Cloud on scaling up or down the hosted {agent}, new containers for the Elastic Agent are started and the old ones are shut down. The old Elastic Agents will show up in the list for 24 hours and will automatically disappear afterwards.
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc:520]: Before I review this, I need to know: Are you talking about the Elastic Agent running Fleet Server? Or an agent that's running to monitor Elastic cloud (I'm assuming there is one)? Or both? I just want to make sure we're clear about what hosted Elastic Agent means.

Comment (ruflin) [reply docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc:520]: The one that has that "Elastic Agent cloud policy" with fleet-server integrations inside assigned.

Screenshot 2021-11-30 at 20 22 30

**Comment by jlind23** [comment]
@ruflin you merged it in 7.16 right? If yes, can we move quickly on this PR?

**Comment by ruflin** [comment]
Correct, this is in 7.16.

File: docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc [hunk 2]
```diff
@@ -511,3 +511,10 @@ installed integrations.
 
 . Under advanced settings, set `windows.advanced.diagnostic.enabled`
 to `false`, and then save the integration.
+
+
+[discrete]
+[[hosted-agent-offline]]
+== Hosted {agent} Offline
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc:518]:
```suggestion
== Hosted {agent} is offline
```

File: docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc (hunk 1 above)
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc:520]:
```suggestion
To scale the {fleet-server} deployment, {ecloud} starts new containers or shuts down old ones when hosted {agent}s are required or no longer needed. The old {agent}s will show in the Agents list for 24 hours then automatically disappear.
```

Comment (dedemorton) [reply docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc:520]: @ruflin I think it's useful to make the connection with Fleet Server here, but not sure if I have the technical details quite right.

Comment (ruflin) [reply docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc]: Make sense to add a notion of fleet-server here to better clarify it. We should make sure we use the same terms / explanations across the docs for it.

**Comment by ruflin** [comment]
@dedemorton I wonder what is more efficient way to push this PR forward. Should I merge it in or you want to take it over? If I take it on? For the backports, can you check if I got the lables right?

@v1v One more place to unify backport labels ;-)

**Comment by v1v** [comment]
> @v1v One more place to unify backport labels ;-)

I don't know if for the `docs` we should use the `backport-v` format. No idea the implications, but it could be  great if we can align those labels with what other teams do.

For the record, I enabled https://github.com/elastic/security-docs and they use the `vx.y.z` format.

I'm happy to enable the mergify automation, if the docs team want to move forward.

**Pull request merged**